hack/vmimage-validate.sh

```diff
@@ -26,3 +26,8 @@ go generate ./... && go run -ldflags "-X main.gitCommit=$(git rev-parse --short=
 
 # Currently there are only 3 logs we want to capture
 mv /tmp/{yum_update_info,yum_check_update,scap-report.html} ${ARTIFACTS:-/tmp}/ || true
+
+if grep -q "rule-result rule-result-fail" ${ARTIFACTS:-/tmp}/scap-report.html; then
+    echo "Some SCAP rules failed. Check report in the build artifacts"
+    exit 1
```

Comment:
/test vmimage

I expect it to fail here.. We will need to merge #1835 to make it pass.

Reply:
Tested locally with some issues with spacing.  This appears to work.

Reply:
> Tested locally with some issues with spacing.

Discussed in slack: it's probably the issue of copy-pasting from Github.

But it made me notice that I used 4 spaces here instead of 2 and I should probably echo into stderr instead of stdout. I'll update the PR shortly.